Changed No to Yes for Create Azure Run As Account

CreepyGnome opened the pull request:
My understanding is that for an Azure Automation that is used for runbooks it needs to be a Run As account and that this section is to help you create just such an account. The warnings for if you set it to No make more sense if Yes is what you are being instructed to set it to.

azurecla commented:
Hi __@fashah__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (fashah). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

CreepyGnome committed:
Changed No to Yes for Create Azure Run As Account

My understanding is that for an Azure Automation that is used for runbooks it needs to be a Run As account and that this section is to help you create just such an account. The warnings for if you set it to No make more sense if Yes is what you are being instructed to set it to.
